### PR TITLE
Don't use unqualified using and update README.dev.md with guidelines

### LINKED ISSUE #431: Apply specific "using" syntax
_updated_
Apply @abelsiqueira's format to all files:

image

----------------------------------------------------------------------------
_original_
### What and Why

Tars suggested using `import` instead of `using`, we should see what difference it makes, the benefits and drawbacks.

### Possible Drawbacks

_No response_

### Related Issues

_No response_

### PR DESCRIPTION
**Pull request by abelsiqueira**
## Describe the changes made in this pull request

using Package should not be used anymore. Instead, explicitly use
using Package: A, B, C to load objects A, B, and C. Also, don't
do that for all objects, only obvious candidates. More information
is included in the README.dev.md new section on code format guidelines.

## List of related issues or pull requests

Closes #431

## Collaboration confirmation

As a contributor I confirm

- [x] I read and followed the instructions in README.dev.md
- [x] The documentation is up to date with the changes introduced in this Pull Request (or NA)
- [x] Tests are passing
- [x] Lint is passing

### CODE REVIEW

**Comment by abelsiqueira** [comment]
This needs to be rebased after #532

File: README.dev.md [hunk 1]
```diff
@@ -402,3 +402,18 @@ installed in your global environment, and run `using Revise` before including
 
 After building, the documentation will be available in the folder `docs/build/`.
 Open the `index.html` file on the browser to see it.
+
+## Code format and guidelines
+
+This section will list the guidelines for code formatting **not enforced** by [JuliaFormatter](.JuliaFormatter.toml).
+We will try to follow these developing and reviewing.
```
Comment (clizbe) [inline README.dev.md:409]:
```suggestion
We will try to follow these during development and reviews.
```

**Review by clizbe — CHANGES_REQUESTED** [review]
Just one little edit, the rest looks great!
Makes the code a bit longer, but I think it actually makes it more understandable.

**Comment by abelsiqueira** [comment]
Thanks for the review, I've updated, and re-requested

**Review by clizbe — APPROVED** [review]
Great!

**Pull request merged**